systori/apps/main/management/commands/generate_swagger_file.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management.base import BaseCommand, CommandError
+from drf_yasg.management.commands import generate_swagger
+
+# TODO Can't override `generate_swagger` unless `main` app is moved before `drf_yasg` in `INSTALLED_APPS`
+# See: https://docs.djangoproject.com/en/2.2/howto/custom-management-commands/#overriding-commands
+class Command(generate_swagger.Command):
+    help = "Generate swagger schema in schema.yaml"
+
+    def handle(self, *args, **options):
+        options["output_file"] = "/app/schema.yaml"
+        options["overwrite"] = True
```

Comment:
can't we remove the line since overwrite isn't working anyways? ... also I'm fine with generate_swagger_file - not sure if it's save to move drf_yasg before main

Reply:
Overwrite is actually working. If you run `python3 manage.py generate_swagger_file` it will overwrite the existing `schema.yaml`. Which is located at `/app/schema.yaml` (in the docker container)